stdlib/Random/test/runtests.jl

```diff
@@ -713,3 +713,60 @@ end
         @test rand((x, 2, 3, 4, 6)) ∈ 1:6
     end
 end
+
+@testset "GLOBAL_RNG" begin
+    local GLOBAL_RNG = Random.GLOBAL_RNG
+    local LOCAL_RNG = Random.get_local_rng()
+    @test VERSION < v"2" # deprecate this in v2
```

Comment:
The `_GLOBAL_RNG` thing is rather clever! Do I get it right that it's only to avoid breaking code and that it's intended to be removed in v2?